homeassistant/components/logbook/__init__.py

```diff
@@ -32,14 +32,17 @@
 from homeassistant.loader import bind_hass
 
 from . import rest_api, websocket_api
-from .const import (
+from .const import (  # noqa: F401
     ATTR_MESSAGE,
     DOMAIN,
     LOGBOOK_ENTITIES_FILTER,
+    LOGBOOK_ENTRY_CONTEXT_ID,
```

Comment:
Should we add also LOGBOOK_ENTRY_STATE and LOGBOOK_ENTRY_WHEN, even though they are not used in the core code (only used in the local `processor` module)

Reply:
We can always add them later if needed. However since they are only used in that one (internal) module, I wouldn't expose them at this point.